libcontainer/intelrdt/mbm.go

```diff
@@ -0,0 +1,114 @@
+// +build linux
+
+package intelrdt
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/sirupsen/logrus"
```

Comment:
Usually standard libraries are split from non-standard packages on the list of imports, proper form of imports have following form:
```
import (
        "bufio"
	"fmt"
	"io"
	"os"
	"path/filepath"

        "github.com/sirupsen/logrus"
)
```
goimport can also help with formatting list of imports, see the section "Formatting Imports" in [this ](https://www.digitalocean.com/community/tutorials/importing-packages-in-go) document.